src/main/java/mate/academy/spring/service/impl/RoleServiceImpl.java

```diff
@@ -0,0 +1,25 @@
+package mate.academy.spring.service.impl;
+
+import mate.academy.spring.dao.RoleDao;
+import mate.academy.spring.model.Role;
+import mate.academy.spring.service.RoleService;
+import org.springframework.stereotype.Service;
+
+@Service
+public class RoleServiceImpl implements RoleService {
+    private final RoleDao roleDao;
+
+    public RoleServiceImpl(RoleDao roleDao) {
+        this.roleDao = roleDao;
+    }
+
+    @Override
+    public Role add(Role role) {
+        return roleDao.add(role);
+    }
+
+    @Override
+    public Role getByName(String roleName) {
+        return roleDao.getByName(roleName);
```

Comment:
```suggestion
        return roleDao.getByName(Role.RoleName.valueOf(roleName));
```
if you will fix getByName on DAO layer you need fix it in service too

Reply:
Same